docs/csharp/programming-guide/strings/index.md

```diff
@@ -64,11 +64,21 @@ A string is an object of type <xref:System.String> whose value is text. Internal
 >  At compile time, verbatim strings are converted to ordinary strings with all the same escape sequences. Therefore, if you view a verbatim string in the debugger watch window, you will see the escape characters that were added by the compiler, not the verbatim version from your source code. For example, the verbatim string @"C:\files.txt" will appear in the watch window as "C:\\\files.txt".  
   
 ## Format Strings  
- A format string is a string whose contents can be determined dynamically at runtime. You create a format string by using the static <xref:System.String.Format%2A> method and embedding placeholders in braces that will be replaced by other values at runtime. The following example uses a format string to output the result of each iteration of a loop:  
-  
- [!code-csharp[csProgGuideStrings#26](../../../csharp/programming-guide/strings/codesnippet/CSharp/index_6.cs)]  
```

Comment:
@hDoubleE could you please delete this code file as well. As the code snippets are moved to the samples repo, there is no reason to keep that file in this repo.

Reply:
So I accidentally deleted index.md and not index_6.cs. Really bad. Trying to figure out how to restore...